services/amo/amo-users.service.js

```diff
@@ -14,6 +14,8 @@ export default class AmoUsers extends BaseAmoService {
     },
   ]
 
+  static _cacheLength = 21600
```

Comment:
@calebcartwright based on your comment in https://github.com/badges/shields/issues/7584#issuecomment-1034296124 I've set downloads/users to `21600` seconds (6 hours). Feel free to tweak it if you want.